docs/source/configuration.rst

```diff
@@ -0,0 +1,388 @@
+=======================
+Configuration Reference
+=======================
+
+This page documents the schema of the YAML configuration file used by the Migrator and the Validator.
+
+The configuration file is a YAML object whose fields are enumerated, preceded by a comment describing their role. All the fields are mandatory except if their documentation starts with “Optional”. All the values of the form ``<xxx>`` are placeholders that should be replaced with your specific settings.
```

Comment:
add something like:
YAML format is whitespace sensitive, make sure you use proper number of spaces before to keep properties on same level.


(breaking whitespace is most common user mistake, I guess this is the weakest(strongest?:-D) spot of python and yaml)

Reply:
Fixed in [4da9711](https://github.com/scylladb/scylla-migrator/pull/166/commits/4da971193facbee455c49626955729ec21a54a4d).